.circleci/config.yml

```diff
@@ -79,4 +79,10 @@ workflows:
             branches:
               ignore: /.*/
             tags:
-              ignore: /^testing-.*/
+              only: /^v.*/
+      - rok8s-scripts/github_release:
```

Comment:
Do we want this to be dependent on the build passing, or is this fine as-is?

Reply:
It will only run if it's a tagged release, so my thought was that would only happen once master is in a good state (i.e. tests are passing). We could add the testing phase to a tagged release also if you want an additional safeguard.

Reply:
Oh I see what you mean. Yeah probably :) I'll make a change.